ChangeLog

```diff
@@ -28,6 +28,7 @@ English Dolibarr ChangeLog
 - Fix: [ bug #1825 ] External agenda: hide/show checkbox doesn't work
 - Fix: [ bug #1790 ] Email form behaves in an unexpected way when pressing Enter key
 - Fix: Bad SEPA xml file creation
+- Fix: [ bug #1856 ] ODT Templates - Templating keys are not always recognized
```

Comment:
Oh snap, didn't know (remember?) it only applied to the develop branch… Thanks for the heads-up!